spec/xctest_list_spec.rb

```diff
@@ -122,6 +122,13 @@ def mock_foreach(filepath_pattern, file_fixture)
       expect(parsed_tests).to eq(["AtomicBoyUITests/testExample2", "AtomicBoyUITests/testExample", "SwiftAtomicBoyUITests/testExample"])
     end
 
+    context 'when given a special test prefix' do
```

Comment:
Just to keep things consistent in the file.

```suggestion
    describe 'when given a special test prefix' do
```

Reply:
Makes sense to me.